.changeset/two-apricots-prove.md

```diff
@@ -0,0 +1,7 @@
+---
+"@ledgerhq/coin-stacks": patch
+"ledger-live-desktop": patch
+"@ledgerhq/live-common": patch
```

Comment:
You cannot use patch anymore and need to use minor outside hotfix branches

Reply:
Oh, already forgot about this morning post, thanks!

Reply:
yes I saw that this morning !
I do the change thx